d/private/rules/binary.bzl

```diff
@@ -1,5 +1,9 @@
 """D test rule for compiling binaries."""
 
+load(
```

Comment:
load it from @rules_cc: https://github.com/bazelbuild/rules_cc/blob/main/cc/find_cc_toolchain.bzl#L114, then it will also fit on one single line :)